packages/design-system/src/styles/components/_StepList.scss

```diff
@@ -28,36 +14,36 @@ $current-step-color: $color-primary !default;
 }
 
 .ds-c-step {
-  color: $color-muted;
+  color: $steplist__color;
   counter-increment: step-counter;
-  margin-bottom: $step-margin;
-  padding-bottom: $step-margin;
+  margin-bottom: $spacer-2;
+  padding-bottom: $spacer-2;
   position: relative;
 
   &::before {
-    border: $step-list-number-border-size solid $color-muted;
-    border-radius: 50%;
+    border: 2px solid $steplist-step__border-color--default;
```

Comment:
Should the `2px` here use a theme variable?

Reply:
I guess we only need to add that customizability if someone actually _wants_ to change it.

Reply:
I'm trying to do mostly color right now with a few spacing variables tossed into the mix for simplicity sake, but yes I definitely see us expanding the variable list in the future when someone wants a change that goes outside the 'norm'